### ISSUE TITLE
The `set-env` command is deprecated and will be disabled soon.

### ISSUE THREAD

**Issue by sblantipodi** [AdoptOpenJDK/install-jdk#10]
Using the adopt install jdk action returns a warning

```
      - name: Set up AdoptOpenJDK 15
        uses: AdoptOpenJDK/install-jdk@v1
        with:
          version: '15'
          architecture: x64
```

this is the warning with some errors.
```
Run AdoptOpenJDK/install-jdk@v1
C:\windows\System32\WindowsPowerShell\v1.0\powershell.exe -NoLogo -Sta -NoProfile -NonInteractive -ExecutionPolicy Unrestricted -Command "$ErrorActionPreference = 'Stop' ; try { Add-Type -AssemblyName System.IO.Compression.FileSystem } catch { } ; [System.IO.Compression.ZipFile]::ExtractToDirectory('D:\a\_temp\5dc80748-a307-4a7c-b574-02c961aae42d', 'D:\a\_temp\temp_746465718')"
Warning: The `add-path` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/
Warning: The `set-env` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/
```

probably you know it but an issue is just a reminder :)
thank you for your great product.

**Comment by TheMrMilchmann** [comment]
Thanks for the report! This should already be fixed on `master` by e0636177ea683530b2171fe8753be99cf6079e40. I'll validate the fix tomorrow (13.10) and publish `1.1.1`.

**Comment by TheMrMilchmann** [comment]
https://github.com/AdoptOpenJDK/install-jdk/commit/e0636177ea683530b2171fe8753be99cf6079e40 only partially fixed the issue as it only upgraded the dependency. I rebuilt the action in 8f7e2a291290b2e3cac6c5dc121d3a27d4f73ad0 and just released `1.1.1` with the fix. If you are using `AdoptOpenJDK/install-jdk@v1`, you should not see this deprecation warning anymore going forward!
